server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -23,6 +26,27 @@
     </div>
   </div>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form id="form-select-characteristic" method="get" novalidate>
+        <div class="govuk-form-group">
+          <label class="govuk-label" for="sort">
```

Comment:
the label is not linked correctly to the select element.
the GDS form component templates are quite fiddly, so generally it's a good idea to use their macros so all the ids and names match up. plus, if GDS update them, we automatically benefit

Reply:
I'm sorting the wrong `for` attribute, well spotted ☺️ 

Regarding the use of the macros, agreed.  Unfortunately, given:
- the desired layout is the one where button is just next to select
- govuk's select macro wraps the select in a `.govuk-form-group` div

This is why we went the HTML route (given is relatively straightforward markup) because this way we can wrap both the select and the button elements in the same `.govuk-form-group` div.